backon/src/blocking_sleep.rs

```diff
@@ -24,13 +24,18 @@ impl<F: Fn(Duration) + 'static> BlockingSleeper for F {
 /// The default implementation of `Sleeper` when no features are enabled.
 ///
 /// It will fail to compile if a containing [`Retry`][crate::Retry] is `.await`ed without calling [`Retry::sleep`][crate::Retry::sleep] to provide a valid sleeper.
-#[cfg(not(feature = "std-blocking-sleep"))]
+#[cfg(all(not(feature = "std-blocking-sleep"), not(feature = "embassy-sleep")))]
 pub type DefaultBlockingSleeper = PleaseEnableAFeatureOrProvideACustomSleeper;
 /// The default implementation of `Sleeper` while feature `std-blocking-sleep` enabled.
 ///
 /// it uses [`std::thread::sleep`].
 #[cfg(feature = "std-blocking-sleep")]
 pub type DefaultBlockingSleeper = StdSleeper;
+/// The default implementation of `Sleeper` while feature `embassy-sleep` enabled.
+///
+/// it uses [`embassy_time::block_for`].
+#[cfg(all(not(feature = "std-blocking-sleep"), feature = "embassy-sleep"))]
+pub type DefaultBlockingSleeper = EmbassySleeper;
```

Comment:
Plesse don't add `EmbassySleeper` as default sleeper. Users will need to specify `EmbassySleeper` instead.

Reply:
Ok